### PR TITLE
Allow a user to include all aggregate report dimensions

### PR DESCRIPTION
**Pull request by wtritch**
We were previously treating dimensions similarly to filters, where if all were selected the front-end was stripping the parameters from the request payload.

### CODE REVIEW

File: webapp/docker-compose.yml [hunk 1]
```diff
@@ -25,109 +25,109 @@ services:
       - 5672:5672
       - 15672:15672
 
-#  config-server:
```
Comment (wtritch) [inline webapp/docker-compose.yml]: unrelated, revert what I assume was an unintended change

File: webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-request.mapper.ts [hunk 2]
```diff
@@ -136,7 +134,7 @@ export class AggregateReportRequestMapper {
             ? options.completenesses
             : query.completenessCodes,
           dimensionTypes: Utils.isNullOrEmpty(query.dimensionTypes)
-            ? options.dimensionTypes
+            ? []
             : query.dimensionTypes,
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-request.mapper.ts]: If the query value is `null` initialize to an empty array.

File: webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-request.mapper.ts [hunk 3]
```diff
@@ -65,9 +66,6 @@ export class AggregateReportRequestMapper {
     if (!equalSize(settings.completenesses, options.completenesses)) {
       query.completenessCodes = settings.completenesses;
     }
-    if (!equalSize(settings.dimensionTypes, options.dimensionTypes)) {
-      query.dimensionTypes = settings.dimensionTypes;
-    }
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-request.mapper.ts]: We just want whatever the settings are, don't strip the values if the user selects them all.

File: webapp/docker-compose.yml (hunk 1 above)
Comment (agorina) [reply webapp/docker-compose.yml]: If this was me, then yes. Sorry and thank you for fixing it!

**Pull request merged**